copyq/__init__.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+"""Access CopyQ clipboard.
+
+Synopsis: <trigger> [filter]"""
+
+#  Copyright (c) 2022 Manuel Schneider
+
+import html
+import json
+import re
+import subprocess
+
+from albert import *
+
+md_iid = "0.5"
+md_version = "1.1"
+md_name = "CopyQ"
+md_description = "Access CopyQ clipboard"
+md_bin_dependencies = ["copyq"]
+
+
+copyq_script_getAll = r"""
+var result=[];
+for ( var i = 0; i < size(); ++i ) {
+    var obj = {};
+    obj.row = i;
+    obj.mimetypes = str(read("?", i)).split("\n");
+    obj.mimetypes.pop();
+    obj.text = str(read(i));
+    result.push(obj);
+}
+JSON.stringify(result);
+"""
+
+copyq_script_getMatches = r"""
+var result=[];
+var match = "%s";
+for ( var i = 0; i < size(); ++i ) {
+    if (str(read(i)).search(new RegExp(match, "i")) !== -1) {
+        var obj = {};
+        obj.row = i;
+        obj.mimetypes = str(read("?", i)).split("\n");
+        obj.mimetypes.pop();
+        obj.text = str(read(i));
+        result.push(obj);
+    }
+}
+JSON.stringify(result);
+"""
+
+
+class Plugin(QueryHandler):
+    def id(self):
+        return __name__
+
+    def name(self):
+        return md_name
+
+    def description(self):
+        return md_description
+
+    def synopsis(self):
```

Comment:
I am not exactly sure but iirc default is exactly filter. May be dropped

Reply:
Without the override of `synopsis` nothing is displayed with an empty query text.

Reply:
Indeed I left it empty. nmv 😅 too much to remember everything